aspnet/mvc/views/forms/sample/final/ViewModels/AddressViewModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿
+namespace FormsTH.ViewModels
+{
+    public class AddressViewModel
+    {
+        public string AddressLine1 { get; set; }
+    }
+
+    public class RegisterViewModel2
+    {
+        public string Email { get; set; }
+
+        public string Password { get; set; }
```

Comment:
So you introduce data annotations and tag helpers in a single stage? Have you confirmed responses do not contain `<input type="text" id="Password" name="Password" />` anywhere in either sample?